kubernetes/resource_kubernetes_persistent_volume_claim_v1_test.go

```diff
@@ -399,114 +402,114 @@ func TestAccKubernetesPersistentVolumeClaim_googleCloud_storageClass(t *testing.
 				},
 				Check: resource.ComposeAggregateTestCheckFunc(
-					testAccCheckKubernetesPersistentVolumeClaimExists("kubernetes_persistent_volume_claim.test", &conf2),
+					testAccCheckKubernetesPersistentVolumeClaimV1Exists("kubernetes_persistent_volume_claim.test", &conf2),
 					resource.TestCheckResourceAttr("kubernetes_persistent_volume_claim.test", "spec.0.resources.0.requests.storage", "2Gi"),
-					testAccCheckKubernetesPersistentVolumeClaimForceNew(&conf1, &conf2, false),
+					testAccCheckKubernetesPersistentVolumeClaimV1ForceNew(&conf1, &conf2, false),
 				),
 			},
 			{ // Minikube specific check -- PVC is recreated when when `resources.limits` is increased.
-				Config: testAccKubernetesPersistentVolumeClaimConfig_updateStorageMinikube(name, "2Gi", "6Gi"),
+				Config: testAccKubernetesPersistentVolumeClaimV1Config_updateStorageMinikube(name, "2Gi", "6Gi"),
 				SkipFunc: func() (bool, error) {
 					isInMinikube, err := isRunningInMinikube()
 					return !isInMinikube, err
 				},
 				Check: resource.ComposeAggregateTestCheckFunc(
-					testAccCheckKubernetesPersistentVolumeClaimExists("kubernetes_persistent_volume_claim.test", &conf2),
+					testAccCheckKubernetesPersistentVolumeClaimV1Exists("kubernetes_persistent_volume_claim.test", &conf2),
 					resource.TestCheckResourceAttr("kubernetes_persistent_volume_claim.test", "metadata.0.name", name),
 					resource.TestCheckResourceAttr("kubernetes_persistent_volume_claim.test", "spec.0.resources.0.limits.storage", "6Gi"),
-					testAccCheckKubernetesPersistentVolumeClaimForceNew(&conf1, &conf2, true),
+					testAccCheckKubernetesPersistentVolumeClaimV1ForceNew(&conf1, &conf2, true),
 				),
 			},
 			{ // Minikube specific check -- PVC is recreated when `resources.requests` is decreased.
-				Config: testAccKubernetesPersistentVolumeClaimConfig_updateStorageMinikube(name, "1Gi", "6Gi"),
+				Config: testAccKubernetesPersistentVolumeClaimV1Config_updateStorageMinikube(name, "1Gi", "6Gi"),
 				SkipFunc: func() (bool, error) {
 					isInMinikube, err := isRunningInMinikube()
 					return !isInMinikube, err
 				},
 				Check: resource.ComposeAggregateTestCheckFunc(
-					testAccCheckKubernetesPersistentVolumeClaimExists("kubernetes_persistent_volume_claim.test", &conf2),
+					testAccCheckKubernetesPersistentVolumeClaimV1Exists("kubernetes_persistent_volume_claim.test", &conf2),
 					resource.TestCheckResourceAttr("kubernetes_persistent_volume_claim.test", "metadata.0.name", name),
 					resource.TestCheckResourceAttr("kubernetes_persistent_volume_claim.test", "spec.0.resources.0.requests.storage", "1Gi"),
-					testAccCheckKubernetesPersistentVolumeClaimForceNew(&conf1, &conf2, true),
+					testAccCheckKubernetesPersistentVolumeClaimV1ForceNew(&conf1, &conf2, true),
 				),
 			},
 		},
 	})
 }
 
-func testAccCheckKubernetesPersistentVolumeClaimDestroy(s *terraform.State) error {
+func testAccCheckKubernetesPersistentVolumeClaimV1Destroy(s *terraform.State) error {
```

Comment:
I cannot leave a comment on a few lines below, so I will put it here. In this function, you validate the resource name `kubernetes_persistent_volume_claim.test`, however, it should be `kubernetes_persistent_volume_claim_v1.test`.